src/Agoda.Analyzers/AgodaCustom/AG0022RemoveSyncMethodFixProvider.cs

```diff
@@ -1,4 +1,5 @@
-﻿using System.Collections.Immutable;
+﻿using System.Collections.Generic;
```

Comment:
Do we need to bring in new `using` but there seems to be no new code added that would require it